src/db/db.ts

```diff
@@ -12,6 +12,58 @@ export async function addAlice() {
   await closeConnection();
 }
 
+export async function createUsers(user: users) {
+  await prisma.users.create({
+    data: {
+      usersname: user.usersname,
+      usersemail: user.usersemail,
+      userspassword: user.userspassword,
+      userstotalscore: null,
+      userslevel: null
```

Comment:
I would suggest to set these to a number instead of null to avoid issues.
Something like the score to 0 and level to 1.